### PR TITLE
NiFi 0.2.1

### PR DESCRIPTION
**Pull request by apiri**
Adding a formula for a version 0.2.1 of Apache NiFi

### CODE REVIEW

File: nifi021.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Nifi < Formula
```
Comment (DomT4) [inline nifi021.rb]: You need to fix the `class` name to match the formula name.

Comment (apiri) [inline nifi021.rb]: Updated and pushed the change.  Thanks!

**Comment by DomT4** [comment]
Just an audit nit left:

```
==> audit problems
homebrew/versions/nifi021:
 * Description is too long. "name: desc" should be less than 80 characters.
Length is calculated as homebrew/versions/nifi021 + desc. (currently 101)
```

**Comment by apiri** [comment]
Patched up, apologies for the audit miss.  Haven't quite worked out the best way to work on taps in terms of formula creations and wasn't able to find any good docs on that area.  Any suggestions as to how this is best accomplished and/or if I am overlooking something obvious?

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/b9fb9eacb1790d7ba290dd480f784e528f5d2e7b. Thank you for your contribution to Homebrew @apiri; we appreciate it! :smiley_cat: 

If you have specific questions on tap things let me know - Happy to answer. The easiest way to check everything is to run `brew audit --strict --online /path/to/formula.rb` on any new submissions.

**Pull request closed without merging**